classes/settings_provider.php

```diff
@@ -291,18 +342,38 @@ public static function add_seb_config_elements(\mod_quiz_mod_form $quizform, \Mo
             }
 
             if (!self::can_manage_setting($name, $quizform->get_context())) {
```

Comment:
This block is not required as we check for can_manage_setting at the start of this function.